sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-cosmosdb/pom.xml

```diff
@@ -24,6 +24,16 @@
       <artifactId>azure-cosmosdb-spring-boot-starter</artifactId>
       <version>2.4.0-beta.1</version> <!-- {x-version-update;com.microsoft.azure:azure-cosmosdb-spring-boot-starter;current} -->
     </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-actuator</artifactId>
+      <version>2.3.3.RELEASE</version>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-web</artifactId>
+      <version>2.3.3.RELEASE</version>
+    </dependency>
   </dependencies>
 
 </project>
```

Comment:
Do we need to add `management.health.azure-cosmos.enabled=true` for `azure-spring-boot-sample-cosmosdb`?

Reply:
it will calculate RUs, so I set the default value `false`, have update the `azure-spring-boot-sample-cosmos` readme, if you want to use the Spring Boot Actuator, you can open it manually

Reply:
> Do we need to add `management.health.azure-cosmos.enabled=true` for `azure-spring-boot-sample-cosmosdb`?

it will calculate RUs, so I set the default value false, have update the azure-spring-boot-sample-cosmos readme, if you want to use the Spring Boot Actuator, you can open it manually